test/AutoDiff/differentiable_attr_type_checking.swift

```diff
@@ -22,7 +22,7 @@ func jvpSimpleJVP(x: Float) -> (Float, ((Float) -> Float)) {
   return (x, { v in v })
 }
 
-@differentiable(wrt: (.1), jvp: jvpWrtSubsetJVP)
+@differentiable(wrt: (y), jvp: jvpWrtSubsetJVP)
```

Comment:
It would make a lot more sense if this looked like
```swift
@differentiable(wrt: y, jvp: jvpWrtSubsetJVP)
```

Reply:
Good point. Added support for single `wrt` parameter without parentheses in https://github.com/apple/swift/pull/22000/commits/e37fec4f5f23135145a8b4c3b25a4d15c8e1a28e.
This enables `wrt: self` for instance methods.